libminifi/test/pcap-tests/CMakeLists.txt

```diff
@@ -29,17 +29,17 @@ FOREACH(testfile ${PCAP_TESTS})
   	target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_BINARY_DIR}/extensions/pcap/pcap++/Dist/header/")
     createTests("${testfilename}")
     if(APPLE)    
-    	target_link_libraries ("${testfilename}" -Wl,-all_load minifi-pcap )
```

Comment:
Good catch! Thanks for making this change!

Reply:
Thanks. The pcap tests no compile, but still not succeed my macOS, but I will open a ticket for that.

Reply:
Did you run them as root?

Reply:
Yes, I did, it fails on the `assert(LogTestController::getInstance().contains("Accepting ") == true && LogTestController::getInstance().contains("because it matches .*") );` assertion.

Reply:
Not surprised, that processor is mostly abandoned and will hopefully be removed in upcoming releases.